lib/cb/utils/api.rb

```diff
@@ -57,10 +57,14 @@ def cb_delete(path, options = {}, &block)
       def execute_http_request(http_method, uri, path, options = {}, &block)
```

Comment:
After your explanation last night (everybody and their brother is mocking the `execute_http_request` method) I agree. Let's do this in another PR, it'll be easier to review this in 2 chunks like you're saying :thumbsup: